internal/cmd/beta/server/create/create.go

```diff
@@ -0,0 +1,338 @@
+			),
+		),
+		PreRun: func(cmd *cobra.Command, _ []string) {
+			bootVolumeSourceId, _ := cmd.Flags().GetString(bootVolumeSourceIdFlag)
+			bootVolumeSourceType, _ := cmd.Flags().GetString(bootVolumeSourceTypeFlag)
+			bootVolumeSize, _ := cmd.Flags().GetInt64(bootVolumeSizeFlag)
+			imageId, _ := cmd.Flags().GetString(imageIdFlag)
+
+			if imageId == "" && bootVolumeSourceId == "" && bootVolumeSourceType == "" {
+				p.Warn("Either Image ID or boot volume flags must be provided.\n")
+			}
+
+			if imageId == "" {
+				err := flags.MarkFlagsRequired(cmd, bootVolumeSourceIdFlag, bootVolumeSourceTypeFlag)
+				cobra.CheckErr(err)
+			}
+
+			if bootVolumeSourceType == "image" {
+				if bootVolumeSize == 0 {
+					p.Warn("Boot volume size must be provided when `source_type` is `image`.\n")
+				}
+				err := cmd.MarkFlagRequired(bootVolumeSizeFlag)
+				cobra.CheckErr(err)
+			}
+
+			if bootVolumeSourceId == "" && bootVolumeSourceType == "" {
+				err := cmd.MarkFlagRequired(imageIdFlag)
+				cobra.CheckErr(err)
+			}
+
+			if imageId != "" && (bootVolumeSourceId != "" || bootVolumeSourceType != "") {
+				p.Warn("Image ID flag cannot be used together with any of the boot volume flags.\n")
+			}
+		},
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, p, cmd)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get project name: %v", err)
+				projectLabel = model.ProjectId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a server for project %q?", projectLabel)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create server : %w", err)
+			}
+			serverId := *resp.Id
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Creating server")
+				_, err = wait.CreateServerWaitHandler(ctx, apiClient, model.ProjectId, serverId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for server creation: %w", err)
+				}
+				s.Stop()
+			}
+
+			return outputResult(p, model, projectLabel, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP(nameFlag, "n", "", "Server name")
+	cmd.Flags().String(machineTypeFlag, "", "Name of the type of the machine for the server. Possible values are documented in [Virtual machine flavors](https://docs.stackit.cloud/stackit/en/virtual-machine-flavors-75137231.html)")
```

Comment:
Is the link correctly rendered and formatted in the terminal? Otherwise we could just do `.. documented in https://docs.stackit.cloud/stackit/en/virtual-machine-flavors-75137231.html`

Reply:
I just put the link instead of formatting. It was shown wrong